examples/dt/dcn/service-values.yaml

```diff
@@ -0,0 +1,261 @@
+          [glance_store]
+          default_backend = az1
+          [az1]
+          rbd_store_ceph_conf = /etc/ceph/az1.conf
+          store_description = "az1 RBD backend"
+          rbd_store_pool = images
+          rbd_store_user = openstack
+          rbd_thin_provisioning = True
+          [az0]
+          rbd_store_ceph_conf = /etc/ceph/az0.conf
+          store_description = "az0 RBD backend"
+          rbd_store_pool = images
+          rbd_store_user = openstack
+          rbd_thin_provisioning = True
+          [az2]
+          rbd_store_ceph_conf = /etc/ceph/az2.conf
+          store_description = "az2 RBD backend"
+          rbd_store_pool = images
+          rbd_store_user = openstack
+          rbd_thin_provisioning = True
+          [image_import_opts]
+          image_import_plugins = ['image_conversion']
+          [image_conversion]
+          output_format = raw
+        networkAttachments:
+          - storage
+        override:
+          service:
+            internal:
+              metadata:
+                annotations:
+                  metallb.universe.tf/address-pool: internalapi
+                  metallb.universe.tf/allow-shared-ip: internalapi
+                  metallb.universe.tf/loadBalancerIPs: 172.17.0.81
+              spec:
+                type: LoadBalancer
+        replicas: 3
+        type: edge
+      az2:
+        customServiceConfig: |
+          [DEFAULT]
+          enabled_import_methods = [web-download,copy-image,glance-direct]
+          enabled_backends = az0:rbd,az1:rbd,az2:rbd
+          [glance_store]
+          default_backend = az2
+          [az2]
+          rbd_store_ceph_conf = /etc/ceph/az2.conf
+          store_description = "az2 RBD backend"
+          rbd_store_pool = images
+          rbd_store_user = openstack
+          rbd_thin_provisioning = True
+          [az0]
+          rbd_store_ceph_conf = /etc/ceph/az0.conf
+          store_description = "az0 RBD backend"
+          rbd_store_pool = images
+          rbd_store_user = openstack
+          rbd_thin_provisioning = True
+          [az1]
+          rbd_store_ceph_conf = /etc/ceph/az1.conf
+          store_description = "az1 RBD backend"
+          rbd_store_pool = images
+          rbd_store_user = openstack
+          rbd_thin_provisioning = True
+          [image_import_opts]
+          image_import_plugins = ['image_conversion']
+          [image_conversion]
+          output_format = raw
+        networkAttachments:
+          - storage
+        override:
+          service:
+            internal:
+              metadata:
+                annotations:
+                  metallb.universe.tf/address-pool: internalapi
+                  metallb.universe.tf/allow-shared-ip: internalapi
+                  metallb.universe.tf/loadBalancerIPs: 172.17.0.82
+              spec:
+                type: LoadBalancer
+        replicas: 3
+        type: edge
+  manila:
+    enabled: true
```

Comment:
disable manila as it's not part of FR1 and probably in Ceph we don't enable `File` (no MDS is deployed, hence this won't work).
Change `enabled: true` to `enabled: false`.
We can keep the config for future Manila deployments if native_cephfs is the target.

Reply:
I've confirmed that manila is disabled in the latest